x/pocketcore/keeper/service_test.go

```diff
@@ -66,25 +66,60 @@ func TestKeeper_HandleRelay(t *testing.T) {
 		t.Fatalf(er.Error())
 	}
 	validRelay.Proof.Signature = hex.EncodeToString(clientSig)
-	defer gock.Off() // Flush pending mocks after test execution
-
-	gock.New("https://www.google.com:443").
-		Post("/").
-		Reply(200).
-		BodyString("bar")
 
 	mockCtx.On("KVStore", keeper.storeKey).Return(ctx.KVStore(keeper.storeKey))
 	mockCtx.On("KVStore", keys["pos"]).Return(ctx.KVStore(keys["pos"]))
 	mockCtx.On("KVStore", keys["params"]).Return(ctx.KVStore(keys["params"]))
 	mockCtx.On("KVStore", keys["application"]).Return(ctx.KVStore(keys["application"]))
-	mockCtx.On("BlockHeight").Return(ctx.BlockHeight())
+	mockCtx.On("BlockHeight").Return(currentBlockHeight)
 	mockCtx.On("PrevCtx", int64(976)).Return(ctx, nil)
 	mockCtx.On("PrevCtx", keeper.GetLatestSessionBlockHeight(mockCtx)).Return(ctx, nil)
 	mockCtx.On("Logger").Return(ctx.Logger())
+	return
+}
+
+func TestKeeper_HandleRelay(t *testing.T) {
+	defer gock.Off() // Flush pending mocks after test execution
 
+	gock.New("https://www.google.com:443").
+		Post("/").
+		Reply(200).
+		BodyString("bar")
+	keeper, mockCtx, validRelay := setupHandleRelayTest(t, 976)
 	resp, err := keeper.HandleRelay(mockCtx, validRelay)
 	assert.Nil(t, err, err)
 	assert.NotNil(t, resp)
 	assert.NotEmpty(t, resp)
 	assert.Equal(t, resp.Response, "bar")
 }
+
+func TestKeeper_HandleRelayWithinTolerance(t *testing.T) {
+	defer gock.Off() // Flush pending mocks after test execution
+
+	gock.New("https://www.google.com:443").
+		Post("/").
+		Reply(200).
+		BodyString("bar")
+	keeper, mockCtx, validRelay := setupHandleRelayTest(t, 977)
+	resp, err := keeper.HandleRelay(mockCtx, validRelay)
+	assert.Nil(t, err, err)
+	assert.NotNil(t, resp)
+	assert.NotEmpty(t, resp)
+	assert.Equal(t, resp.Response, "bar")
+}
+
+func TestKeeper_HandleRelayOutsideTolerance(t *testing.T) {
+	// Each session block is 25 blocks each
```

Comment:
Why?
```suggestion
	// Each session is 25 blocks
```

Reply:
I also asked myself the same question when making these tests and was surprised myself.

https://github.com/pokt-network/pocket-core/blob/629a38891850507c4374bdd8d75cbdfd343d469c/x/nodes/types/params.go#L22

It's part of the default values, which I assume is then the default values for the tests since we use global variables instead of something mockable in these tests

Reply:
Let's reference `params.DefaultSessionBlocktime` in the comment